packages/fxa-settings/src/components/Settings/index.tsx

```diff
@@ -23,19 +19,7 @@ export const Settings = () => {
   const modalHeaderId = 'modal-header-verify-email';
   const modalDescId = 'modal-desc-verify-email';
 
-  if (loading) {
-    return (
-      <LoadingSpinner className="bg-grey-20 flex items-center flex-col justify-center h-screen select-none" />
-    );
-  }
-
-  if (error) {
-    return <AppErrorDialog data-testid="error-dialog" {...{ error }} />;
-  }
-
-  const account: accountData = data.account;
-  const primaryEmail = account.emails.find((email) => email.isPrimary)!;
-  // const secondaryEmails = account.emails.filter((email) => !email.isPrimary);
+  const primaryEmail = account.emails.find((email: any) => email.isPrimary)!;
```

Comment:
Because `account` is typed isn't the interface of `email` here implicitly `{ email: string; isPrimary: boolean; verified: boolean; }`? If not I'd recommend we type it since it's likely we'll be using this type of object a lot throughout this project.

Reply:
Yes, this should be taken out. I added this when I was trying to get fancy separating primary email and secondary emails and the type was no longer inferred so good catch.